### ISSUE TITLE
wordlists for australian languages (sutton and walsh)

### ISSUE THREAD

**Issue by LinguList** [clld/concepticon-data#17]
I don't have access to this book, but it could be useful to include it, since it seems to offer new concept lists for Australian region:

Sutton, P. and M. Walsh (1987). Wordlist for Australian Languages. Canberra:
Australian Institute of Aboriginal Studies.

**Comment by nthieberger** [comment]
I have a correspondence table of several wordlists of Australian languages that I used to show degree of overlap. I'm interested in building it out to include more lists to show how much we can compare earlier sources that were based on these lists. 
Nick Thieberger. 2016. Daisy Bates in the digital world. In Austin, Peter K., Harold Koch and Jane Simpson (eds) Language, land and song: Studies in honour of Luise Hercus. London: EL Publishing. (http://www.elpublishing.org/itempage/2008)

**Comment by LinguList** [comment]
Sounds very interesting. By "correspondence table" you mean sound correspondences? Have you by any chance seen our experiments on inferring and annotating correspondence patters? You find a draft [here](https://doi.org/10.1101/434621). It sounds like self-advertising, but the cool thing about this approach is that we can even predictin non-existing cognates in the descendant languages with some degree of certainty, which is some automatic proof of hte regularity of sound change, in my opinion...

**Comment by LinguList** [comment]
Very cool, your paper, @nthieberger, in fact, all the lists you compare there should ideally all be linked to Concepticon, in which case the whole comparison could be done also with other lists. Also interesting to compare the history of evolution here, since these lists, as far as I can see, are not deeply based on Swadesh, right?

**Comment by nthieberger** [comment]
Right, these all precede Swadesh, by 50 or more years. The correspondences
are simply an alignment of the senses provided in each list so see how much
they have in common. This could be extended to a number of other Australian
lists and would be interesting from a historiographic point of view, seeing
how much was learned about the languages over time and what were
appropriate things to look for in the languages.

**Comment by nthieberger** [comment]
OK, so what will be involved to get these lists aligned? Can I get someone
here to go through and link to existing concepts, and then what about new
ones?
What is the mechanism for doing this?

Nick

**Comment by LinguList** [comment]
The basic format is a spreadsheet per list of some number (for internal numbering) and the English gloss:

```
NUMBER | ENGLISH
--- | --- 
1 | hand / arm
```

This should render the source. Then, we need the bibliography in bibtex format, a statement of two to three sentences explaining the list, and that's basically all. 

We would then automatically pre-link the data:

```
NUMBER | ENGLISH | CONCEPTICON_ID | CONCEPTICON_GLOSS
--- | --- | --- | ---
1 | hand | 1 | HAND
```

And then this is being corrected. See [here](https://calc.hypotheses.org/1820) and the follow-up. 

It is for now enough if you send us, e.g., one list in the format above, and we send our automatic mappings back.

**Comment by xrotwang** [comment]
@nthieberger I think the spreadsheet you used for your comparison in the linked article should be enough to get us started. If you could send this my way, I could prepare the basic skeleton for a Concepticon contribution.

**Comment by nthieberger** [comment]
OK, here's the spreadsheet.
It includes four lists, Sutton & Walsh (S&W), Bates 1904, Curr , Capell
wordlist

S&W is
Sutton, Peter, & Michael Walsh. 1979. Revised Linguistic Fieldwork Manual
for Australia. AIAS New Series no.8. Canberra: Australian Institute of
Aboriginal Studies.

Bates is the questionnaire she devised and sent out in 1904 (
http://bates.org.au/quest.html)

Curr is from:  The Australian race : its origin, languages, customs, place
of landing in Australia and the routes by which it spread itself over the
continent
by Curr, Edward Micklethwaite, 1820-1889
https://archive.org/details/cu31924026093827/page/n6

Capell is from : Capell, Arthur. 1963. Linguistic survey of Australia.
Canberra: AIAS.

Hope this makes sense but let me know if not.
Nick

**Comment by xrotwang** [comment]
@nthieberger attaching excel to github issues does not work, AFAICT. Maybe it'd be best to send via email.

**Comment by xrotwang** [comment]
Note: Sutton and Walsh is https://catalogue.nla.gov.au/Record/2893467

**Comment by xrotwang** [comment]
@nthieberger as a first pass, I tried to extract just the Sutton & Walsh list from the excel sheet. It seems the list has identifiers which are concatenated from `S&W id` and `S&W codes`, right? There are quite a few cases, though, where only one gloss is given for a number of these identifiers. E.g. `Elasmobranchii` has IDs `58K` through `82K`. This seems to happen mostly for biological cover terms. What should we do here?

- Could (and should) we try to find more specific glosses? (I.e. are more specific glosses in the book?)
- Should we collapse multiple IDs into one concept - as you did in the spreadsheet?

**Comment by xrotwang** [comment]
I also couldn't make much sense of the `S&W codes`. Do these codes provide a useful categorization of the concepts, or are they merely the letters of an arbitrary alphanumeric identifier?

**Comment by nthieberger** [comment]
You can probably just discard the codes as they are internal to the
wordlist?
